docs/splunk-apis.md

```diff
@@ -0,0 +1,32 @@
+# Splunk APIs
+
+### 1. Retrieve Ingested Data:
+Ingest data in JSON format. Supports reading json data from file.
+
+```
+GET /api/search
+POST /api/search
+```
+
+Parameters:
+- `searchText`: The actual search query using Splunk syntax.
+- `indexName:` Specifies the index (like "traces") to search data from.
+
```

Comment:
you need to add startepock, endepoch , query langyage,
see the example that you added below and a short line describing what each of these fields mean